benchmark/torch/mappo/simple_model.py

```diff
@@ -0,0 +1,94 @@
+#   Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# modified from https://github.com/marlbenchmark/on-policy
+
+import parl
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class SimpleModel(parl.Model):
+    def __init__(self, obs_dim, cent_obs_dim, act_dim):
+        super(SimpleModel, self).__init__()
+        self.act_dim = act_dim
+
+        self.actor = Actor(obs_dim, self.act_dim)
+        self.critic = Critic(cent_obs_dim)
+
+    def policy(self, obs, available_actions=None, deterministic=False):
```

Comment:
"deterministic" is not accessed

Reply:
Please check if `available_actions` is required

Reply:
available_actions is required in simple_speaker_listener.py